assets/blocks/reader-registration/view.js

```diff
@@ -20,12 +20,25 @@ import './style.scss';
 			}
 		} );
 
+		const startLoginFlow = () => {
+			messageElement.innerHTML = '';
+			submitElement.disabled = true;
+			container.classList.add( 'newspack-registration--in-progress' );
+		};
+
 		const endLoginFlow = ( message, status, data ) => {
+			messageElement.innerHTML = '';
+			container.classList.remove( 'newspack-registration--in-progress' );
+
+			if ( ! message || ! status ) {
+				return;
+			}
 			const messageNode = document.createElement( 'p' );
 			messageNode.innerHTML = message;
```

Comment:
`innerHTML` is a potential security concern—we should use `textContent` instead.

Reply:
Done in 5867cfa2bef7bff21be6dbd49c1a77fc5c7f72de

Reply:
Looks good, thanks!